// korcen/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.special = exports.sexual = exports.politics = exports.foreign = exports.parents = exports.minor = exports.belittle = exports.race = exports.general = exports.check = void 0;
var checkBadLang_1 = require("./checkBadLang");
Object.defineProperty(exports, "check", { enumerable: true, get: function () { return checkBadLang_1.check; } });
var checkGeneral_1 = require("./checkGeneral");
Object.defineProperty(exports, "general", { enumerable: true, get: function () { return checkGeneral_1.general; } });
var checkRace_1 = require("./checkRace");
Object.defineProperty(exports, "race", { enumerable: true, get: function () { return checkRace_1.race; } });
var checkBelittle_1 = require("./checkBelittle");
Object.defineProperty(exports, "belittle", { enumerable: true, get: function () { return checkBelittle_1.belittle; } });
var checkMinor_1 = require("./checkMinor");
Object.defineProperty(exports, "minor", { enumerable: true, get: function () { return checkMinor_1.minor; } });
var checkParents_1 = require("./checkParents");
Object.defineProperty(exports, "parents", { enumerable: true, get: function () { return checkParents_1.parents; } });
var checkForeign_1 = require("./checkForeign");
Object.defineProperty(exports, "foreign", { enumerable: true, get: function () { return checkForeign_1.foreign; } });
var checkPolitics_1 = require("./checkPolitics");
Object.defineProperty(exports, "politics", { enumerable: true, get: function () { return checkPolitics_1.politics; } });
var checkSexual_1 = require("./checkSexual");
Object.defineProperty(exports, "sexual", { enumerable: true, get: function () { return checkSexual_1.sexual; } });
var checkSpecial_1 = require("./checkSpecial");
Object.defineProperty(exports, "special", { enumerable: true, get: function () { return checkSpecial_1.special; } });
